### DISCUSSION TITLE
Cannot not render macros

### DISCUSSION THREAD

**Discussion by NFanoe** [fralau/mkdocs-macros-plugin#251] [Q&A]
I want to exclude the rendering of macros on certain pages that include math and typescript code blocks. However, neither the 
```
---
render_macros: false
---
```
yaml header nor the 
`{% raw %} {% endraw %}` construct work. When I mkdocs serve, the log shows errors, and the pages do not render.
What am I doing wrong?

**Comment by NFanoe** [comment]
Hold on - I guess I was wrong, sorry about that. I had to re-generate the whole thing, then both methods worked for me.
Sorry about the fuss.

**Comment by fralau** [comment]
Glad it was solved for you!
